toolchest_client/api/query.py

```diff
@@ -55,12 +55,18 @@ def __init__(self, stored_output=None, is_async=False, pipeline_segment_instance
                 self.PIPELINE_SEGMENT_INSTANCE_URL,
                 "status",
             ])
+            self.OUTPUT_STREAM_URL = "/".join([
+                self.PIPELINE_SEGMENT_INSTANCE_URL,
+                "output-stream",
+            ])
```

Comment:
These URLs are getting a bit unwieldy, and `OUTPUT_STREAM_URL` is defined three times and only used once.

Cleanup is outside the scope of this PR, but as an intermediate step maybe move this to `get_output_stream_latest_lines`?